concepticondata/concepticon.tsv

```diff
@@ -1128,8 +1128,8 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 1127	SEND	Motion	To make something (an object, a message) go from one place to another.	Action/Process	
 1128	ORDER	Basic actions and technology	To order someone or several persons by a superior authority to do something, or to arrange something in a tidy manner.	Action/Process	
 1129	PAINFUL	Sense perception	Full of pain; causing uneasiness or distress, either physical or mental.	Property	
-1130	WE (EXCLUSIVE)	Kinship	exclusive “we”: I and at least one other person, but not the addressed person (you) (pronoun 1st person plural without 2nd person)	Other	
-1131	WE (INCLUSIVE)	Kinship	inclusive “we”: I and at least one other person, including the addressed person(s) (pronoun 1st person plural with 2nd person)	Other	
+1130	WE (EXCLUSIVE)	Kinship	Exclusive “we”: I and at least one other person, but not the addressed person (you) (pronoun 1st person plural without 2nd person)	Other	
+1131	WE (INCLUSIVE)	Kinship	Inclusive “we”: I and at least one other person, including the addressed person(s) (pronoun 1st person plural with 2nd person)	Other	
```

Comment:
Perhaps you could modify WE (EXCLUSIVE) so that it also says "but not the addressed person(s)" (instead of just one person)?

Reply:
I think WE (EXCLUSIVE) is mainly used if one talks to another person about something that she is doing without the addressed person (singular). For example, "we go to the concert (but not you)".

Reply:
I thought it works analog to WE (INCLUSIVE) which is why I argue for equal definitions. In both cases, you may be talking to more than one person and include or exclude that group which is being spoken or referred to. 
Then you'd have "We go to the concert (but not you, Dirk and Linda, to which I am currently talking)" and "We go to the concert (including you, Dirk and Linda, to which I am currently talking)".
It's not a big thing at all and we can leave it as it is, just wanted to explain where I'm coming from.

Reply:
Sure, I get the idea. The tricky part is that "you" is also under-specified (see https://en.wikipedia.org/wiki/Clusivity).